test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -198,6 +199,9 @@ def _addError(self, test, err):
             self.logger.info('ERROR\n')
         elif self.dots:
             self.logger.info('E')
+        if not hasattr(test, 'test'):
+            if not (issubclass(type(test.test), PBSTestSuite)):
+                raise KeyboardInterrupt
```

Comment:
As discussed offline, PTLTestResult class is only meant for logging results, there should be no errors raise from it.